proof-market/glossary.mdx

```diff
@@ -0,0 +1,45 @@
+# Glossary
+
+This page defines the key terms used in Proof Market. 
+
+## Circuit
+
+A circuit is an algebraic representation of an algorithm consisting of a series of polynomial equations. 
+
+The [**zkLLVM compiler**](../zkllvm/overview/what-is-zkllvm) allows anyone to write a circuit in either C++ or Rust. Circuit code can be compiled into an intermediate representation (IR) in `.ll` format, which can then be used to generate a statement.
+
+Note that a circuit is typically provided by a proof requester.
+
+## Statement
+
+A statement is the primary object of trade in Proof Market.
+
+A statement contains a circuit and additional metadata for trading. To publish a statement, create a [**circuit IR**](../zkllvm/getting-started/compiling-a-circuit), and [**perform an API request**](./application-developers/tutorials/managing-statements).
+
+Similarly to circuits, statements are typically generated by proof requesters. 
+
+## Proof requester
+
+An entity (an application or a user) that wants to use zero-knowledge proofs to verify something on the blockhain.
+
+## Proof producer
+
+A Proof Market participant that generates proofs that have been asked for by proof requesters.
+
+## Proposal
+
+A proposal is an offer made by a proof producer to generate a proof for a specific statement and for a specific price.
+
+## Request
+
+A request is a formalized call to generate a proof for a specific statement with given public inputs and for a specific reward. Whenever public inputs change, a new request has to be posted.
+
+:::info[Request statuses]
+
+A request may have the following statuses.
+
+* `CREATED`: the initial status for any request before it is matched with a proposal.
+* `PENDING`: the status assigned after a request is matched with a proposal. At this stage, Proof Market awaits a proof producer to generate a proof.
+* `DONE`: the final status for any successful request. It is assigned after proof is provided by a proof producer.
```

Comment:
There is one more, which is `ERROR` is something bad happens

Reply:
I see, added